core/jazz_services/test/test.js

```diff
@@ -74,6 +74,7 @@ describe('platform_services', function() {
       },
       "body" : {
         "description" : "g0nna_GET_a-L!tt1e_we!rd",
+        "deployment_accounts" : [{"accountId":"12345","region":["us-east-1"],"provider":"aws","primary":true}],
```

Comment:
I think region is not an array

Reply:
Actually region is coming as an array as one account can have multiple regions. So to maintain the uniformity, I made it as an array here in test file.